### PR TITLE
projects:ad7124_iio: Release AD7124 IIO application

### PR DESCRIPTION
**Pull request by D-Disha**
1) Release AD7124 IIO Application.
2) Update mbedignore to add entry for ad7124_iio

### CODE REVIEW

**Comment by jansunil** [comment]
@D-Disha Data capture scripts folder seems to be missing

**Review by jansunil — DISMISSED** [review]

File: projects/ad7124_iio/app/app_config.c [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+		return ret;
+	}
+#endif
+
+	return 0;
+}
+
+#if (ACTIVE_PLATFORM == MBED_PLATFORM)
+/**
+ * @brief 	Initialize the lvgl timer peripheral
```
Comment (SaikiranGudla) [inline projects/ad7124_iio/app/app_config.c:249]: nit: Can we format the extra space in the doxygen descriptions of all the functions? This was seen in other files too.

File: projects/ad7124_iio/app/app_config.h [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+// **** Note for User on selection of Active Device ****//
+/* Define the device type here from the list of below device type defines
+ * (one at a time. Defining more than one device can result into compile error).
+ * e.g. #define DEV_AD7142_4= -> This will make AD7124-4 as an active device.
+ * The active device is default set to AD7124-4 if device type is not defined.
+ * */
+//#define DEV_AD7124_4
+
+#if defined(DEV_AD7124_4)
+#define DEVICE_NAME			"DEV_AD7124_4"
```
Comment (SaikiranGudla) [inline projects/ad7124_iio/app/app_config.h:74]: nit: Shall we format the spacing here (74 to 91 lines) for uniformity?

**Review by SaikiranGudla — DISMISSED** [review]
Some minor comments. LGTM overall.

**Review by jansunil — APPROVED** [review]

**Review by SaikiranGudla — APPROVED** [review]

**Pull request merged**